tests/foreman/endtoend/test_cli_endtoend.py

```diff
@@ -335,17 +329,18 @@ def test_positive_cli_end_to_end(fake_manifest_is_set, rhel6_contenthost, defaul
     # step 2.18: Provision a client
     # TODO this isn't provisioning through satellite as intended
     # Note it wasn't well before the change that added this todo
-    rhel6_contenthost.install_katello_ca(default_sat)
-    # Register client with foreman server using act keys
-    rhel6_contenthost.register_contenthost(org['label'], activation_key['name'])
-    assert rhel6_contenthost.subscribed
-    # Install rpm on client
-    package_name = 'python-kitchen'
-    result = rhel6_contenthost.execute(f'yum install -y {package_name}')
-    assert result.status == 0
-    # Verify that the package is installed by querying it
-    result = rhel6_contenthost.run(f'rpm -q {package_name}')
-    assert result.status == 0
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}) as client:
```

Comment:
Same here, use `rhel7_contenthost` fixture instead of directly calling broker.

Reply:
@ogajduse Updated 👍